project-fob/Controllers/HomeController.cs

```diff
@@ -79,135 +79,109 @@ public ActionResult MeetingPageHost(string attendeePassword, string hostPassword
 
             db.Meeting.Add(meet);
             HttpContext.Session.Set("meetingid", Encoding.ASCII.GetBytes(meet.MeetingId));
-            //Session["meetingid"] = meet.MeetingId;
 
             db.Fob.Add(new Fob(meet));
             db.SaveChanges();
 
             @ViewBag.title = "Meeting Id: " + meet.MeetingId;
-            //TODO generae QR code for the meeting and display it on its own page (add something to host to get this if lost)
+            ViewBag.meetingid = meet.MeetingId;
 
             return View();
         }
 
-
-        //Join Meeting                                          //will this string be empty or null ?
         public ActionResult MeetingPageUser(string meetingId, string password)
-        { //The password will require different way to send it because atm it is visible
+        {
+            if (password == null)
             {
-                if (password == null)
-                {
-                    password = "";
-                }
-                meetingId = meetingId.ToUpper();
-                Meeting meet = db.Meeting.SingleOrDefault(m => m.MeetingId == meetingId);
+                password = "";
+            }
```

Comment:
To make the password empty string instead of null, you could use `password = password  ?? "";`